checkpoint/bin/checkpoint_vector_models_download.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+usage () {
+    echo $"
+Usage: $0 
+        [-h vector-model-host-url] 
+        [-o vector-model-target-path] 
+    <checkpoint_path>
+"
+}
+
+BIN="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+CHECKPOINT_ROOT=$( dirname "$BIN" )
+VECTOR_MODEL_TARGET_PATH=${CHECKPOINT_ROOT}/vector_models
+VECTOR_MODEL_HOST_URL=http://mentorpal-vector-models.s3-website-us-east-1.amazonaws.com
```

Comment:
Do we need an API key or AWS account to access this?

Reply:
no, I made it publicly readable. Really we could download it from google drive, but because of how google drive works, you need to install a pkg to download large files/can't just use `wget`, so I I figured easier for us to just host ourselves